com.unity.netcode.gameobjects/Editor/com.unity.netcode.editor.asmdef

```diff
@@ -7,5 +7,19 @@
     ],
     "includePlatforms": [
         "Editor"
-    ]
+    ],
+    "excludePlatforms": [],
+    "allowUnsafeCode": false,
+    "overrideReferences": false,
+    "precompiledReferences": [],
+    "autoReferenced": true,
+    "defineConstraints": [],
+    "versionDefines": [
+        {
+            "name": "com.unity.multiplayer.tools",
+            "expression": "",
+            "define": "MULTIPLAYER_TOOLS"
+        }
```

Comment:
is this coming up because you have MP TOOLS package installed locally?

Reply:
No, this is an intentional change, it's present in other assemblies but we didn't have any tools integration in this assembly until now.